tinyagentos/projects/notes_store.py

```
@@ -0,0 +1,95 @@
+                return None
+            keys = [d[0] for d in cur.description]
+            return dict(zip(keys, row))
+
+    async def list_notes(self, project_id: str) -> list[dict]:
+        async with self._db.execute(
+            "SELECT * FROM project_notes WHERE project_id = ? ORDER BY created_at DESC",
+            (project_id,),
+        ) as cur:
+            rows = await cur.fetchall()
+            keys = [d[0] for d in cur.description]
+            return [dict(zip(keys, r)) for r in rows]
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)